docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+version: '3.2'
+
+services:
+  roach-ui:
+    image: cockroachdb/cockroach
+    command: start --insecure
```

Comment:
this should be changed in production, right? I mean to set up secure connection, right?